include/trick/montecarlo_c_intf.h

```diff
@@ -23,6 +23,18 @@ void mc_set_enabled(int enabled);
  */
 int mc_get_enabled(void);
 
+/**
+ * @relates Trick::MonteCarlo
+ * @copydoc MonteCarlo::set_processing_input_for_MC
```

Comment:
I'm not sure what this does, may need to add something to documentation before merging or change this line.